recommenders/models/rbm/rbm.py

```diff
@@ -273,14 +256,14 @@ def init_parameters(self):
 
             self.bv = tf.compat.v1.get_variable(
                 "v_bias",
-                [1, self.Nvisible],
+                [1, self.n_visible],
                 initializer=tf.compat.v1.zeros_initializer(),
                 dtype="float32",
             )
 
             self.bh = tf.compat.v1.get_variable(
                 "h_bias",
-                [1, self.Nhidden],
+                [1, self.n_hidden],
```

Comment:
there is an extra input in:
```
    def losses(self, vv):
        """Loss functions.
        Args:
            v (tf.Tensor, float32): empirical input
            v_k (tf.Tensor, float32): sampled visible units at step k
        Returns:
            object:
            - Objective function of Contrastive divergence: the difference between the free energy clamped on the data (v) and the model Free energy (v_k).
        """
        with tf.compat.v1.variable_scope("losses"):
            obj = self.free_energy(vv) - self.free_energy(self.v_k)
        return obj
```
`v_k (tf.Tensor, float32): sampled visible units at step k` is not an argument. 

Also, typically the description of the loss function is added at the beginning of the function: 
```
Objective function of Contrastive divergence: the difference between the free energy clamped on the data (v) and the model Free energy (v_k).
```

Reply:
Done.